robottelo/ui/locators/base.py

```diff
@@ -297,6 +298,42 @@
         By.XPATH,
         ("//a[contains(@href, 'compute_resources/') and "
          "contains(@href, 'vms/') and contains(., '%s')]")),
+    "resource.get_by_name": (
+        By.XPATH,
+        "//a[not(contains(@href, 'search')) and contains(@href, '%s')]"),
+
+    # Locators under compute-resources vm tab.
+    "resource.vm_power_button": (
+        By.XPATH,
+        "//table[contains(@id, 'DataTables')]//a[contains(@data-id, '%s')"
+        "and .='%s']/../../td[6]//a"),
```

Comment:
Is any other way to avoid indexes here as it is very unstable way to find the elements?